bind_uploader.go

```diff
@@ -292,12 +292,8 @@ func supportedArrayBind(nv *driver.NamedValue) bool {
 		// TODO SNOW-176486 variant, object, array
 
 		// Support for bulk array binding insertion using []interface{}
-		nvValue := reflect.ValueOf(nv)
-		if nvValue.Kind() == reflect.Ptr {
-			value := reflect.Indirect(reflect.ValueOf(nv.Value))
-			if value.Kind() == reflect.Slice || value.Kind() == reflect.Struct {
-				return true
-			}
+		if isInterfaceArrayBinding(nv.Value) {
```

Comment:
I don't believe we've made any changes to this method; seems like the merge conflict detector is operating on the file level?

Reply:
I think that the way we have been rebasing forces us to rebase their changes onto their changes sometimes which is very confusing. I had to manually go through and handpick our changes onto their current up-to-date version of this file